lib/control-connection.js

```diff
@@ -141,7 +245,7 @@ ControlConnection.prototype.getConnection = function (callback) {
  * it's the first time that trying to retrieve host information
  * @param {Function} callback
  */
-ControlConnection.prototype.initOnConnection = function (firstTime, callback) {
+ControlConnection.prototype.refreshOnConnection = function (firstTime, callback) {
   var c = this.connection;
   var self = this;
   self.log('info', 'Connection acquired, refreshing nodes list');
```

Comment:
oh geez, just realized we're doing a 'once' instead of 'on'.  Well that explains it :)